pkg/gitrepo/repo.go

```diff
@@ -203,17 +207,35 @@ func (r *Repo) ResolveVersion(ctx context.Context, ref string) (string, error) {
 
 			// Sort commits in the queue by commit date in
 			// descending order to find the most recent tag first.
-			sort.Sort(commitSlice(queue))
+			sort.Slice(queue, func(i, j int) bool { return queue[i].Committer.When.After(queue[j].Committer.When) })
```

Comment:
This is unrelated but since I'm changing this I thought using `sort.Slice` is more comprehensive and doesn't require jumping around.